test/extended/testdata/builds/volumes/csi-sharedresourcerolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: authorization.openshift.io/v1
+kind: RoleBinding
+metadata:
+  name: shared-resource-secret
+  namespace: build-volumes-1
+roleRef:
+  name: shared-resource-secret
+  namespace: build-volume
+subjects:
+- kind: ServiceAccount
+  name: default
```

Comment:
builds use the 'builder' service account  ... change default to builder

Reply:
Updated